### PR TITLE
Update README.md of AWS S3 Exporter with example of setting AWS Crede…

### PR DESCRIPTION
**Pull request by jek-bao-choo**
…ntial for OpenTelemetry Collector Helm Chart

**Description:** <Describe what has changed.>

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by jek-bao-choo** [comment]
Hi @atoulme @dmitryax would you be able to accept my PR? Thanks.

**Comment by Frapschen** [comment]
@jek-bao-choo  please sign CLA

**Comment by jek-bao-choo** [comment]
hi @Frapschen Thank you for reaching out. It's signed.

**Review by MovieStoreGuy — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

File: exporter/awss3exporter/README.md [hunk 1]
```diff
@@ -85,3 +85,13 @@ This exporter follows default credential resolution for the
 
 Follow the [guidelines](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) for the
 credential configuration.
+
+### OpenTelemetry Collector Helm Chart for Kubernetes
+For example, when using OpenTelemetry Collector Helm Chart you could use `extraEnvs` in the values.yaml.
+```yaml
+extraEnvs:
+- name: AWS_ACCESS_KEY_ID
+  value: "< YOUR AWS ACCESS KEY KEY >"
```
Comment (dmitryax) [inline exporter/awss3exporter/README.md:94]:
```suggestion
  value: "< YOUR AWS ACCESS KEY >"
```

**Pull request merged**